agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -2240,7 +2240,7 @@ bool slave_thread::handle_cmdu_ap_manager_message(Socket *sd,
 
         auto tuple_supported_channels = response->supported_channels_list(0);
```

Comment:
`preferred_channels = supported_channels`  is confusing 
if we consider ACS report as preferred channels , why not change the CMDU parameter as well .
the code will be more understandable:

```
auto tuple_preferred_channels = response->preferred_channels_list(0);
        std::copy_n(&std::get<1>(tuple_supported_channels), message::SUPPORTED_CHANNELS_LENGTH,
                    hostap_params.preferred_channels);
```

Reply:
Because that will mean an overhaul of changes throughtout the code base to change all `supported_channels` to `preferred_channels`

Reply:
So this should be the first commit in this PR - rename supported_channels to preferred_channels. What you did here is simply unacceptable and VERY confusing.

Reply:
Added a commit that refactors all the ACS related variables and messages to use `preferred_channels`